go.mod

```diff
@@ -8,7 +8,7 @@ go 1.12
 
 require (
 	bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
-	github.com/Azure/azure-sdk-for-go v21.4.0+incompatible
+	github.com/Azure/azure-sdk-for-go v29.0.0+incompatible
```

Comment:
Have you run `./hack/lint-dependencies.sh` for the vendor updates? Last time I'm trying to do this, a lot of conflicts are introduced. 

See https://github.com/kubernetes/kubernetes/pull/76656#issuecomment-484352825

Reply:
we reverted back to v21.4